queries/tiger/highlights.scm

```diff
@@ -32,21 +33,16 @@
   "while"
 ] @repeat
 
-[
- "new"
-] @constructor
+"new" @constructor
```

Comment:
probably `@operator.keyword` like in C++. `@constructor` was applied rather to identifiers constructing rather to keywords/symbols helping with construction (approximately `@function`)